clients/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/NessieUtils.scala

```diff
@@ -101,7 +101,7 @@ object NessieUtils {
     val catalogConf = SparkSession.active.sparkContext.conf
       .getAllWithPrefix(s"spark.sql.catalog.$catalogName.")
       .toMap
-    HttpClientBuilder
+    NessieClient
```

Comment:
this is coming from the iceberg-spark3-runtime dependency and not from the `NessieClient` version that we currently have